pkg/hostagent/event_poster.go

```diff
@@ -109,7 +109,7 @@ func (agent *HostAgent) processPacketEvent(packetEvent PacketEvent, currTime tim
 	if agent.shouldIgnore(packetEvent, currTime) {
 		return nil
 	}
-	var srcEventPosted bool = false
+	var eventDestination *v1.Pod = nil
```

Comment:
nit: you can assume it is nil unless you set the pointer

Reply:
ok, will remove the init.